src/app/modules/group/components/user-list/user-list.component.ts

```diff
@@ -14,37 +15,31 @@ import { GetGroupMembersService, Member } from '../../http-services/get-group-me
 export class UserListComponent implements OnChanges, OnDestroy {
 
   @Input() group? : Group;
-  state: 'loading' | 'error' | 'ready' = 'loading';
+
+  state: Ready<Member[]> | Fetching | FetchError = fetchingState();
 
   currentSort: string[] = [];
 
   members: Member[] = [];
```

Comment:
I do not really like storing twice the same data in the instance method. A simple solution I see is to store as "state" on line 28, the state.tag.

Reply:
I kept it so the table is still visible during the sorting, otherwise the whole table is replace by the progress spinner.
I don't see another solution, I have to keep the data stored somewhere during the fetching